src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameReaderTests.cs

```diff
@@ -27,7 +28,14 @@ public void TestSignaturesV2_3_X()
             DataFrameReader dfr = _spark.Read();
 
             Assert.IsType<DataFrameReader>(dfr.Format("json"));
-
+            
+            Assert.IsType<DataFrameReader>(
+                dfr.Schema(
+                    new StructType(new[]
+                    {
+                        new StructField("age", new IntegerType()),
```

Comment:
How is this working without `StructField` having JVMReference?

Reply:
I am not so sure why only `StructType` has JVMReference works. But I will explore more of PySpark's implementaion for now.